api/openapi-spec/swagger.json

```diff
@@ -1420,7 +1420,8 @@
         }
       },
       "required": [
-        "replicas"
+        "replicas",
+        "availableReplicas"
```

Comment:
adding a new field marked as required is not backwards compatible... this should be marked optional

Reply:
Can this be fixed reverted in the next patch version?

Reply:
yes, @ravisantoshgudimetla is opening a PR and will pick it back